features/steps/steps.py

```diff
@@ -186,6 +202,18 @@ def step_impl(context, field, values):
 
     context.applicable = getattr(context, 'applicable', True) and applicable
 
+
+@given('The {entity} is a part of another {other_entity} through the relationship {relationship}')
```

Comment:
I think this is a good attempt at formalizing the statement we were discussing earlier on separating the direction (is part of / has parts) and the type of the relationship. Should we make the direction a variable too?

Reply:
I am unsure if there is any other relationship that would mean *being part of* / *has parts* in IFC. Is there another `{relationship}` that could even be a candidate for reuse of this rule?

Reply:
Discussed during meeting:

We rephrase the statement to make relationship the subject of the sentence, that way we can use from and to as a generic way to establish the direction of the relationship.

```
> Given An IfcWall
>   And A relationship IfcRelAggregates (from IfcWall) to IfcBuildingElementPart
  
> Given An IfcBuildingElementPart
>   And A relationship IfcRelAggregates (to IfcBuildingElementPart) from IfcWall
     
     (from)               (to)
    Relating             Related
IfcWall <- IfcRelAggregates -> IfcBuildingElementPart
```

Reply:
Implemented.